src/components/molecules/UiPhoneNumber/_internal/UiPhoneNumberPrefix/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <UiDropdown
+    v-model="defaultCountryCode"
+    :text="prefixButtonText"
+    :items="prefixCodes"
+    class="ui-phone-number-prefix"
+    :popover-attrs="{ class: 'ui-phone-number-prefix__popover' }"
+  >
+    <template
+      #toggle="{
+        text,
+        isOpen,
+        toggleHandler,
+      }"
+    >
+      <UiPhoneNumberPrefixToggle
+        :is-open="isOpen"
+        @click="toggleHandler"
+      >
+        {{ text }}
+      </UiPhoneNumberPrefixToggle>
+    </template>
+    <div
+      role="radiogroup"
+      class="ui-dropdown__items"
+    >
+      <UiDropdownItem
```

Comment:
WHY we need `<UiDropdownItem/>` instead using only  `{{ country }} ({{ code }})`?
```
<template slotname> 
{{ country }} ({{ code }})
<template>
```
I see two changes 
```
:aria-setsize="prefixCodes.length"
:aria-posinset="position + 1"
```
maybe it should be included in UiDropdown directly? WDYT?